sdk/lib/atl/atlsimpstr.h

```diff
@@ -358,6 +358,10 @@ class CSimpleStringT
         return GetData()->nDataLength;
     }
 
+    PXSTR GetString() throw()
```

Comment:
Does **MS** ATL have such const + non-const overloaded versions?

Reply:
MS ATL is using const versions only:

![image](https://github.com/reactos/reactos/assets/2107452/a89bed7b-bd5f-4dd3-bf6a-a7518e91a6a0)

Reply:
And in traits:

![image](https://github.com/reactos/reactos/assets/2107452/4a56233f-8d9b-4f5e-94f0-b1ff8e039b26)

Reply:
Personally I think it would be just preferrable to follow the MS ATL there.

Reply:
Cc @JoachimHenze